api/Element.json

```diff
@@ -5906,7 +5926,7 @@
               "version_added": true
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
Let's revert this too, Safari was in fact the first browser to ship a fullscreen API, but webkit-prefixed.